src/realm/collection.hpp

```diff
@@ -176,7 +176,7 @@ class CollectionBase : public Collection {
 
     /// Returns true if the accessor is in the attached state. By default, this
     /// checks if the owning object is still valid.
-    virtual bool is_attached() const
+    virtual bool is_attached() const noexcept
     {
         return get_obj().is_valid();
```

Comment:
nit: `⁣get_obj` can also be marked `noexcept`.

Reply:
???